### PR TITLE
Adds `pulser` required module

### LINKED ISSUE #157: `neutral_atom_compile` error, `pulser` module requirement
`pip` installing `pulser` and attempting an import in the jupyter notebook doesn't solve the issue (note: `pulser` is a SuperstaQ requirement):

pulser-css-error

### PR DESCRIPTION
**Pull request by dowusu-antwi**
Fixes #157. Adds `pulser` module (and installs correct version of `numpy`). Example notebook requires running `pip install -r pulse-requirements.txt`.

css-pulser

### CODE REVIEW

**Review by singular-value — COMMENTED** [review]
Is the idea that we would add to the README that users who want pulse support should `pip install pulser`?

File: examples/neutral_atom_compile.ipynb [hunk 1]
```diff
@@ -8,6 +8,7 @@
    "outputs": [],
    "source": [
     "import cirq\n",
+    "import pulser\n",
```
Comment (singular-value) [inline examples/neutral_atom_compile.ipynb:11]: would this nb still work without this import?

Comment (dowusu-antwi) [reply examples/neutral_atom_compile.ipynb]: Yes -- fixed.

**Review by vtomole — CHANGES_REQUESTED** [review]
Change the requirements name to `neutral-atom-requirements.txt`. In the README, what order do we want to have the users run the `requirements.txts` if they want to do neutral atom stuff?

**Comment by dowusu-antwi** [comment]
> Change the requirements name to `neutral-atom-requirements.txt`. In the README, what order do we want to have the users run the `requirements.txts` if they want to do neutral atom stuff?

Done.

> Change the requirements name to `neutral-atom-requirements.txt`. In the README, what order do we want to have the users run the `requirements.txts` if they want to do neutral atom stuff?

Done (added optional `pip install` for `neutral-atom-requirements`.txt).

**Review by vtomole — APPROVED** [review]
LGTM

**Pull request merged**